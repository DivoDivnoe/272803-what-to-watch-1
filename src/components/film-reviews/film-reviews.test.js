import React from 'react';
import {create} from 'react-test-renderer';
import FilmReviews from './film-reviews.jsx';

const mock = {
  reviews: [
    {
      id: 11,
      comment: `Yohoho`,
      date: ``,
      user: {
        id: 2,
        name: `user`,
      }
    }
  ],
};

describe(`FilmReviews component`, () => {
  it(`renders correctly`, () => {
    const {reviews} = mock;

    const tree = create(
        <FilmReviews reviews={reviews} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
