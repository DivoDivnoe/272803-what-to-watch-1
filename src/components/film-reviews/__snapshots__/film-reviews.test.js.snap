// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmReviews component renders correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Yohoho
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            user
          </cite>
          <time
            className="review__date"
            dateTime=""
          >
            Invalid Date NaN, NaN
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      />
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  />
</div>
`;
